tests/in/interpolate.wgsl

```diff
@@ -0,0 +1,12 @@
+struct FragmentInput {
+  [[location(0), interpolate(flat)]] flat : f32;
```

Comment:
let's have some integer attributes as well

Reply:
What sorts of attributes do you mean, exactly? Do you just mean, we should be mixing in other sorts of attributes in the list?

Reply:
The test you added only uses `f32` type. I'm just suggesting to have a few members of this struct to be `i32` or `u32` instead.

Reply:
Oh - of course. I had forgotten entirely about the right side of the colon...